routes.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+const app = express();
+const port = 7000;
+
+const data = [
+  {
+    ID:1,
+    FoodItem1:"Chocolate",
+    FoodItem2:"Ketchup",
+    WeirdCombos:"Chocolate and Ketchup",
+    Description:"Eat dipped chocolate in ketchup."
+  },
+];
+
+app.use(express.json());
+
+app.get('/', (req, res) => {
+    res.json(data)
+});
+
+app.post('/', (req, res) => {
+    const create = req.body;
+    data.push(create);
+    res.json(data)
+})
+
+app.put('/:index', (req, res) => {
```

Comment:
Add error handling for the CRUD operations. For example, check if the index provided in the PUT and DELETE requests is valid and within the array bounds. If it's not, return an error response. [important]